pinot-core/src/main/java/org/apache/pinot/core/query/reduce/BrokerReduceService.java

```diff
@@ -59,7 +59,8 @@ public BrokerReduceService(PinotConfiguration config) {
   }
 
   public BrokerResponseNative reduceOnDataTable(BrokerRequest brokerRequest, BrokerRequest serverBrokerRequest,
-      Map<ServerRoutingInstance, DataTable> dataTableMap, long reduceTimeOutMs, BrokerMetrics brokerMetrics) {
+                                                Map<ServerRoutingInstance, DataTable> dataTableMap,
+                                                long reduceTimeOutMs, BrokerMetrics brokerMetrics) {
```

Comment:
nit: here you are still aligning in an unexpected way

Reply:
Reverted.